lib/csv_import.rb

```diff
@@ -84,8 +84,13 @@ def import_with_ldap(user_data)
   # tries to save using only the csv data. This method will return
   # false if the data specified in the csv is invalid on the user model.
   def attempt_save_with_csv_data?(user_data)
-    # use username if using cas, email otherwise
-    user_data[:username] = user_data[ENV['CAS_AUTH'] ? :username : :email]
+    if ENV['CAS_AUTH']
+      # set the cas login
+      user_data[:cas_login] = user_data[:username]
```

Comment:
Did this work? I guess in principle the next time the app started the user's CAS login would be copied over from the `cas_login` column to the `username` column, but immediately following the import this user would actually have a nil `username` (I think). I'll test this manually.